test/fixtures/comprehensions/await/actual.js

```diff
@@ -1,4 +1,4 @@
 f() -/>
-  <- [for x of arr:
+  <- [for now x of arr:
```

Comment:
nope yeah this is good. I guess there was a bug before with auto-const for `for-of`...